components/plan-build/bin/hab-plan-build.sh

```diff
@@ -716,10 +716,18 @@ _resolve_dependency() {
 _install_dependency() {
     local dep="${1}"
     if [[ -z "${NO_INSTALL_DEPS:-}" ]]; then
-    $HAB_BIN install -u $HAB_BLDR_URL --channel $HAB_BLDR_CHANNEL "$dep" || {
+
+    # Enable --ignore-local if invoked with HAB_FEAT_IGNORE_LOCAL in
+    # the environment, set to either "true" or "TRUE" (features are
+    # not currently enabled by the mere presence of an environment variable)
+    if [[ "${HAB_FEAT_IGNORE_LOCAL:-}" = "true" ||
+              "${HAB_FEAT_IGNORE_LOCAL:-}" = "TRUE" ]]; then
```

Comment:
I think it's worth considering the value of considering any non-empty value here to be true. This still won't work with `t`, `T`, `1`, `yes`, etc. The code for doing it the other way is nice and simple:
```bash
    if [[ ${HAB_FEAT_IGNORE_LOCAL:-} ]]; then
```

Reply:
I completely agree. This reflects the current behavior of how feature flags are enabled, though, so I wanted to stay consistent. I'm gonna file an issue to fix that up more broadly.

Reply:
@baumanj See #5275